unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -57,6 +58,17 @@ private void Awake()
 
     public override void Populate(ChatEntryModel chatEntryModel)
     {
+        PopulateTask(chatEntryModel);
+    }
+
+    public async UniTask PopulateTask(ChatEntryModel chatEntryModel)
+    {
+        // Due to a TMPro bug in Unity 2020 LTS we have to wait several frames before setting the body.text to avoid a
+        // client crash. More info at https://github.com/decentraland/unity-renderer/pull/2345#issuecomment-1155753538
+        await UniTask.NextFrame();
+        await UniTask.NextFrame();
+        await UniTask.NextFrame();
```

Comment:
I'm really concerned on how reliable this is. We can merbe but we should test it intensively.

Reply:
Yes, actually with a 2-frame wait the crash is avoided, I put a 3-frame wait just in case, although in all my tests 2 was enough to avoid the crash.